src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
Looks like it is defined in `System.Private.Uri`.

[`public static readonly string SchemeDelimiter = "://";`](https://github.com/dotnet/corefx/blob/7df6b74cc4aeebc86e76b5c7901a59fa7fa793f6/src/System.Private.Uri/src/System/Uri.cs#L29)

Since it is a static field you can't add it to the compatibility code as is, probably best to still define it as a const or readonly field though.